tests/test_coordinator.py

```diff
@@ -82,7 +82,7 @@ def test_local_coordinator(control_setting_name):
     assert c_read.max_value == c.max_value
     assert c_read.timestamp == c.timestamp
 
-
+@pytest.mark.timeout(10)
```

Comment:
All these kept failing for me and hanging indefinitely. Added this decorator to throw a timeout. I think all network requests should have a timeout (as should all potentially hanging operations)